src/components/views/project/projectActionCard/AdminActions.tsx

```diff
@@ -61,7 +66,7 @@ export const AdminActions = () => {
 			type: OptionType.ITEM,
 			icon: <IconVerifiedBadge16 />,
 			cb: () => setShowVerificationModal(true),
-			disabled: verified,
+			disabled: isVerificationDisabled,
 		},
 		{
 			label: capitalizeAllWords(
```

Comment:
Let's create a variable for each option (e.g. `editProjectOption`, `verifyYourProject` ,...) and at line 102 create the `activeOptoins` by explicitly mentioning active options and not by index
Change this 
	const activeOptions = isActive ? options : [options[0], options[2]];
With this
	const activeOptions = isActive ? options : [editProjectOption, activeDeactiveOption];

Reply:
I looked at the `DropDown` component, the `disabled` property has `isHidden` behaviour in fact. Thus, it's better only set the property value per option based on the `isAcive` value. I would suggest even changing the `disabled` property name.